@scripts/slick.remotemodel.js

```diff
@@ -0,0 +1,135 @@
+"use strict";
```

Comment:
Should these files be checked in or should we be ignoring the directory?

Reply:
I think since the examples also use the @scripts directory we should probably commit those (not sure where those are hosted).